gemfiles/rails6.1.gemfile

```diff
@@ -1,4 +1,5 @@
 eval_gemfile 'common.rb'
 
 gem 'rails', '~> 6.1.0'
-gem 'genspec', github: 'bquorning/genspec', branch: 'rails-6'
+gem 'genspec', github: 'zendesk/genspec', branch: 'rails-7'
+gem 'sorted_set'
```

Comment:
I think we need to add `sorted_set` as a runtime dependency, and also `require "sorted_set"` lib/curly/presenter.rb